source/material_model/simple.cc

```diff
@@ -323,14 +283,8 @@ namespace aspect
       }
       prm.leave_subsection();
     }
-  }
-}
 
-// explicit instantiations
-namespace aspect
-{
-  namespace MaterialModel
-  {
```

Comment:
Please leave these few lines in there. They're not strictly necessary, but they visually separate things that are logically separate.